cypress/support/pages/ContactUsPage.js

```diff
@@ -0,0 +1,59 @@
+var emailLocator = 'input[id="email"]';
+var orderReferenceLocator = 'input[id="id_order"]';
+var messageLocator = 'textarea[id="message"]';
+var submitMessageLocator = 'button[id="submitMessage"]';
+var pageURL = 'http://automationpractice.com/index.php?controller=contact';
+var email = 'email@email.com'
+var invalidEmail = 'emailemail.com'
+
+class ContactUsPage {
+  constructor () {}
+
+  	openContactUs () {
+    	cy.visit(pageURL)
+    	cy.url().should('eq', pageURL)
+  	}
+
+  	selectSubjectHeading (destination) {
+  		const subjects = ["Customer service", "Webmaster"];
+  		if (destination == null) {
+  			destination = subjects[Math.floor(Math.random())];
+  		}
+  		cy.get('select').select(destination)
+  	}
+
+  	enterEmail () {
+  		cy.get(emailLocator).type(email)
+  	}
+
+  	enterMessage () {
+  		cy.get(messageLocator).type("This is a test message.")	
+  	}
+
+  	clickSend () {
+  		cy.get(submitMessageLocator).click()
+  	}
+
+  	messageSuccessfullySubmitted () {
+  		cy.contains("Your message has been successfully sent to our team.").should('be.visible')
```

Comment:
Would suggest calling out this UI string in a variable as was done for the locators.